docs/db/changelog/changesets/powerauth-java-server/1.7.x/20240115-add-columns-fido2.xml

```diff
@@ -45,11 +45,7 @@
         </addColumn>
     </changeSet>
 
-    <changeSet id="3" logicalFilePath="powerauth-java-server/1.5.x/20230323-add-columns-fido2" author="Petr Dvorak">
-        <dropNotNullConstraint tableName="pa_operation" columnName="user_id"/>
-    </changeSet>
-
-    <changeSet id="4" logicalFilePath="powerauth-java-server/1.5.x/20230323-add-columns-fido2" author="Petr Dvořák">
+    <changeSet id="3" logicalFilePath="powerauth-java-server/1.5.x/20230323-add-columns-fido2" author="Petr Dvořák">
```

Comment:
```suggestion
    <changeSet id="3" logicalFilePath="powerauth-java-server/1.7.x/20230323-add-columns-fido2" author="Petr Dvořák">
```
This change should also be included in the migration guide. It may be generated into SQL script just before the release or right now, it depends.

Reply:
Yes, see: https://github.com/wultra/powerauth-server/issues/1247